clusterloader2/testing/load/modules/scheduler-throughput.yaml

```diff
@@ -0,0 +1,117 @@
+## Input Params
+{{$namespaces := .namespaces}}
```

Comment:
why not use .namespaces directly?
similarly for other variables

Reply:
I see that this is convention started by Matt, probably to state variables explicitly, so I retract my reservations.

Reply:
that was my motivation behind that, thanks Jacek